web/src/js/ducks/flows.js

```diff
@@ -46,6 +48,11 @@ export default function reducer(state = defaultState, action) {
                 ...state,
                 highlight: action.highlight
             }
+        case SET_SORT:
```

Comment:
The idea is that the view is already sorted in the state. The only actions that modify `view` are `UPDATE_FLOWS`, `SET_FILTER` and `SET_SORT`. 
- `For UPDATE_FLOWS` we can just pass `makeSortFn(...)` as an additional argument to `updateViewList` (this is already implemented, see [here](https://github.com/mitmproxy/mitmproxy/blob/7cb7d9ad32c40cff9ceb0e28a5123960fed3638e/web/src/js/ducks/utils/view.js#L77))
- Same for `SET_FILTER`/`updateViewFilter`
- We probably just want to add `updateViewSort()` in `utils/view` for `SET_SORT` which does exactly the same as `updateViewFilter`, except that it creates a copy of the existing list in the first line instead of calling `.filter(...)` on all flows.